virtool/oidc/utils.py

```diff
@@ -0,0 +1,65 @@
+import json
+import os
+from dataclasses import dataclass, asdict
+
+import jwt
+from aiohttp.web import Application
+from jose.jwt import decode, get_unverified_header
+
+
+@dataclass
+class JWKArgs:
+    kty: str
+    kid: str
+    use: str
+    n: str
+    e: str
+
+
+async def validate_token(token: jwt, app: Application) -> dict:
```

Comment:
Convention in Virtool:
```suggestion
async def validate_token(app: Application, token: jwt) -> dict:
```